content/blog/2020-02-24-applications-for-2020-intern-program-are-now-open/index.Rmarkdown

```diff
@@ -0,0 +1,95 @@
+---
+title: "Applications for 2020 Intern Program Are Now Open"
+author: "greg"
+date: '2020-02-20'
+draft: yes
+photo:
+  author: Grant Durr
+  url: https://unsplash.com/photos/1N7yzqjCLes
+publishDate: '2020-02-24'
+publishdate: '2020-02-24'
+slug: applications-for-2020-intern-program-are-now-open
+tags: internship
+description: |
+  Want to apply to this year's summer intern program? Here's how.
+---
+
+As [we announced last year][preview],
+RStudio's summer intern program will run for a third time in 2020,
+and applications are now open.
+Five people will work with an RStudio mentor full-time for 10 weeks starting in May 2020.
+Everyone is welcome to apply:
```

Comment:
We should comment that the interns should be prepared to effectively work remotely. From previous announcement: 

> RStudio is a geographically distributed team which means you can be based anywhere in the United States (we hope to expand the program to support interns in other countries next year). This means that unless you are based in Boston or Seattle, you will be working 100% remotely, though you will meet with your mentor regularly online, and we will pay for you to travel to one face-to-face work sprint with them.

In the past, we have also stressed that the need to be able to _demonstrate_ that they can effectively work in GitHub: 

> To demonstrate these skills, your application needs to include a link to a package, Shiny app, or data analysis repository on GitHub. It’s OK if you create something specifically for this application: we just want to know that you’re already familiar with the mechanics of collaborative development in R.

I would say a good number of applicants in the past two years had little to no GH projects or activity. We should dissuade these folks from applying.

Reply:
@topepo Thanks for the reminder about remote work - paragraph added. The application form asks them about Git repos - in discussion last Friday we decided not to make it an absolute requirement.